config.lua.dist

```diff
@@ -101,11 +101,13 @@ rateExp = 5
 rateSkill = 3
 rateLoot = 2
 rateMagic = 3
-rateSpawn = 1
 
 -- Monsters
+-- spawnIntervalRate control the rate of monster spawning interval
+-- 100 = default, 50 = half spawntime, 200 = double spawntime
```

Comment:
its percentage so add comment, so people know. 100 is 100%, 50 is 50% in this case.